firmware/octopi_firmware_v2/main_controller_teensy41/main_controller_teensy41.ino

```diff
@@ -1959,23 +2163,29 @@ void loop() {
   // keep checking position process at suitable frequence
   if(us_since_last_check_position > interval_check_position) {
 	  us_since_last_check_position = 0;
-
 	  // check if commanded position has been reached
-  	  if (X_commanded_movement_in_progress && tmc4361A_currentPosition(&tmc4361[x]) == X_commanded_target_position && !is_homing_X && !tmc4361A_isRunning(&tmc4361[x], stage_PID_enabled[x])) // homing is handled separately
+    if (X_commanded_movement_in_progress && tmc4361A_currentPosition(&tmc4361[x]) == X_commanded_target_position && !is_homing_X && !tmc4361A_isRunning(&tmc4361[x], stage_PID_enabled[x])) // homing is handled separately
```

Comment:
I think some funkiness happened with white space and bracket lines in this section.

Reply:
@veerwang please fix indentation

Reply:
OK